packages/wrangler/src/r2/lifecycle.ts

```diff
@@ -0,0 +1,412 @@
+import { readConfig } from "../config";
+import { confirm, multiselect, prompt } from "../dialogs";
+import { UserError } from "../errors";
+import { logger } from "../logger";
+import { readFileSync } from "../parse";
+import { printWranglerBanner } from "../update-check";
+import { requireAuth } from "../user";
+import formatLabelledValues from "../utils/render-labelled-values";
+import {
+	formatActionDescription,
+	getLifecycleRules,
+	isNonNegativeNumber,
+	isValidDate,
+	putLifecycleRules,
+	tableFromLifecycleRulesResponse,
+} from "./helpers";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { LifecycleRule } from "./helpers";
+
+export function ListOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("bucket", {
+			describe: "The name of the R2 bucket to list lifecycle rules for",
+			type: "string",
+			demandOption: true,
+		})
+		.option("jurisdiction", {
+			describe: "The jurisdiction where the bucket exists",
```

Comment:
nit:
```suggestion
			describe: "The jurisdiction where the R2 bucket exists",
```

Reply:
This is consistent with other jurisdiction option descriptions